src/AdvanceSteelNodes/Util/Property.cs

```diff
@@ -91,6 +91,17 @@ public static ASProperty ByNameAndValue(string propertyType, object propertyValu
       return objectToModifiy;
     }
 
+    private static ClassTypeFilter createFilterObject(List<int> objectFilters)
```

Comment:
this function is also implemented in AdvanceSteel.Nodes.Selection.ObjectSelection . We should avoid duplicates